conf/igenomes.config

```diff
@@ -340,6 +340,7 @@ params {
       gtf         = "${params.igenomes_base}/Danio_rerio/UCSC/danRer10/Annotation/Genes/genes.gtf"
       bed12       = "${params.igenomes_base}/Danio_rerio/UCSC/danRer10/Annotation/Genes/genes.bed"
       mito_name   = "chrM"
+      macs_gsize  = "1.37e9"
```

Comment:
Do we actually use igenomes in any way in this pipeline? If not, this could be removed, right?

Reply:
you mean the whole igenomes.config file? igenomes can be used for the reference for host read removal

Reply:
Oh right, no than lets keep it as is, ofc!